_docs/setup/kubernetes/ansible-install.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Installing with Ansible
+overview: Instructions on using the included Ansible playbook to perform installation.
+
+order: 40
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+The Ansible scenario defined within this project will allow you to : 
+
+- Install `Istio` Distribution and set the path of the `istioctl` go client (if you execute the command manually)
+- Deploy Istio on Openshift or Kubernetes by specifying different parameters (version, enable auth, deploy bookinfo, ...)
+- Specify the addons to be deployed such as `Grafana`, `Prometheus`, `Servicegraph`, `Zipkin` or `Jaeger`
+
+## Prerequisites
+
+- [Ansible 2.4](http://docs.ansible.com/ansible/latest/intro_installation.html)
+
+Refer to the Ansible Installation Doc on how to install Ansible on your machine.
+To use [Minishift](https://docs.openshift.org/latest/minishift/command-ref/minishift_start.html) or [Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) for local clusters, please refer to their respective documentation. 
+
+## Execution
+
+The role assumes that the user :
+- Can access a Kubernetes or Openshift cluster via `kubectl` or `oc` respectively and is authenticated against the cluster. 
+- Has the `admin` role on the OpenShift platform
+
+Remark : Furthermore the minimum Kubernetes version that is compatible is `1.7.0` (`3.7.0` is the corresponding OpenShift version).   
+
+**Important**: All invocations of the Ansible playbooks need to take place at the `install/ansible` path of the project.
+Failing to do so will result in unexpected errors 
+
+The simplest execution command looks like the following:
+ 
+```bash
+ansible-playbook main.yml
+```
+
+Remarks:
+- The role tries it's best to be idempotent, so running the playbook multiple times should be have the same effect as running it a single time.   
+- The default parameters that apply to this role can be found in `istio/defaults/main.yml`.
+
+The full list of configurable parameters is as follows:
+
+| Parameter | Description | Values |
+| --- | --- | --- |
+| `cluster_flavour` | Defines whether the target cluster is a Kubernetes or an Openshift cluster. | Valid values are `k8s` and `ocp` (default) |
+| `github_api_token` | The API token used for authentication when calling the GitHub API | Any valid GitHub API token or empty (default) |
+| `cmd_path` | Can be used when the user does not have the `oc` or `kubectl` binary on the PATH | Defaults to expecting the binary is on the path | 
+| `istio.release_tag_name` | Should be a valid Istio release version. If left empty, the latest Istio release will be installed | `0.2.12`, `0.3.0`, `0.4.0` (default) |
```

Comment:
since the various deployment tools are in dev or pre-alpha stage, I think it really only makes sense to pull the latest version until the codebases firm up a bit for the various deployment tools. 

Also this isn't your fault (this review has taken awhile) however, 0.5.0 is the current release.

Reply:
I also opened https://github.com/istio/istio/pull/3602 which changes the playbook itself to use the latest version